apps/v1/consts.go

```diff
@@ -0,0 +1,71 @@
+package v1
+
+const (
+	DeploymentStatusReasonAnnotation  = "openshift.io/deployment.status-reason"
+	DeploymentIgnorePodAnnotation     = "deploy.openshift.io/deployer-pod.ignore"
+	DeploymentPodAnnotation           = "openshift.io/deployer-pod.name"
+	DeployerPodCreatedAtAnnotation    = "openshift.io/deployer-pod.created-at"
+	DeployerPodStartedAtAnnotation    = "openshift.io/deployer-pod.started-at"
+	DeployerPodCompletedAtAnnotation  = "openshift.io/deployer-pod.completed-at"
+	DeploymentReplicasAnnotation      = "openshift.io/deployment.replicas"
```

Comment:
This one seems internal, I am not even sure from the top of my head why we have it

Reply:
well, I don't see it used by anything ;-)

Reply:
I see it being set, not used

Reply:
https://github.com/openshift/origin/blob/610fb485858ddab97d8dc35b08ddb4b809b47780/pkg/apps/util/deployment.go#L100
https://github.com/openshift/origin/blob/610fb485858ddab97d8dc35b08ddb4b809b47780/pkg/apps/controller/deployer/deployer_controller.go#L320

Reply:
Wow, isn't that set incorrectly?

Add a TODO to revisit its meaning.

Reply:
I don't think we need that annotation at all, nor should upstream (they are actively using similar one). If there is a change and you see the DC template and the RC it seems pretty straight forward to distinguish a scale update from config update. 

Don't add it, I am gonna drop it's occurences.